### PR TITLE
Bump tf to 2.6.0 to avoid security issue

### PR DESCRIPTION
**Pull request by jmaslek**
`pred` still works on my end after this update

### CODE REVIEW

**Comment by jmaslek** [comment]
need to bump pywin back to 300

File: pyproject.toml [hunk 1]
```diff
@@ -94,7 +94,7 @@ pyupgrade = "^2.11.0"
 Sphinx = "4.1.1"
 
 [tool.poetry.extras]
-prediction = ["tensorflow", "pmdarima", "transformers", "torch"]
+prediction = ["tensorflow", "pmdarima"]
```
Comment (aia) [inline pyproject.toml:97]: Are we removing transformers and torch?

File: requirements-full.txt [hunk 2]
```diff
@@ -162,15 +163,15 @@ python-dotenv==0.19.0; python_version >= "3.5"
 pytrends==4.7.3
 pytz==2021.1; python_full_version >= "3.6.8" and python_version >= "3.6" and python_full_version < "4.0.0"
 pyupgrade==2.18.2; python_full_version >= "3.6.1"
-pywin32==300; sys_platform == "win32" and python_version >= "3.6"
+pywin32==301; sys_platform == "win32" and python_version >= "3.6"
```
Comment (aia) [inline requirements-full.txt:166]: you need to regenerate requirements-full.txt

File: requirements.txt [hunk 3]
```diff
@@ -148,15 +148,15 @@ python-dotenv==0.19.0; python_version >= "3.5"
 pytrends==4.7.3
 pytz==2021.1; python_full_version >= "3.6.8" and python_version >= "3.6" and python_full_version < "4.0.0"
 pyupgrade==2.18.2; python_full_version >= "3.6.1"
-pywin32==300; sys_platform == "win32" and python_version >= "3.6"
+pywin32==301; sys_platform == "win32" and python_version >= "3.6"
```
Comment (aia) [inline requirements.txt:151]: And requirements.txt

File: pyproject.toml (hunk 1 above)
Comment (jmaslek) [reply pyproject.toml:97]: These have been long gone (whenever I added vader for twitter)

File: requirements-full.txt (hunk 2 above)
Comment (jmaslek) [reply requirements-full.txt:166]: I regenerated after running poetry update them, but now I can't get poetry to revert pywin32 `Unable to find installation candidates for pywin32 (300)`.  

pip install pywin32==300 also fails, so I may just close this for now.

Comment (aia) [reply requirements-full.txt:166]: I can do a big dependency update PR.

**Pull request closed without merging**